qemu/tests/rh_kernel_update.py

```diff
@@ -330,6 +329,13 @@ def upgrade_guest_pkgs(session, pkg, debuginfo=False,
             if params.get("install_rpm_from_local") == "yes":
                 rpm_install_func = copy_and_install_rpm
 
+            kernel_deps_pkgs = params.get("kernel_deps_pkgs").split()
+            if kernel_deps_pkgs:
+                for pkg in kernel_deps_pkgs:
+                    arch = params.get("arch_dracut",
```

Comment:
On 08/04/2015 02:42 PM, Shuping Cui wrote:
>
> In qemu/tests/rh_kernel_update.py 
> <https://github.com/autotest/tp-qemu/pull/419#discussion_r36161174>:
>
> > @@ -330,6 +329,13 @@ def upgrade_guest_pkgs(session, pkg, debuginfo=False,
> >              if params.get("install_rpm_from_local") == "yes":
> >                  rpm_install_func = copy_and_install_rpm
> >
> > +            kernel_deps_pkgs = params.get("kernel_deps_pkgs").split()
> > +            if kernel_deps_pkgs:
> > +                for pkg in kernel_deps_pkgs:
> > +                    arch = params.get("arch_dracut",
>
> Yes, but for dracut on RHEL.6 guest, need use '--arch=noarch' get brew 
> link, like this 'brew latest-pkg xxx dracut --arch=noarch --paths', so 
> still need arch_dracut param.
>
We can set all depend packages use param like "upgrade_pkgs", and 
pkg_arch should be configurable not limited to arch_dracut.


> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/autotest/tp-qemu/pull/419/files#r36161174>.
>